README.md

```diff
@@ -33,7 +33,7 @@ Table of contents:
 - [Quickly deploy it on K8s and get started](docs/quick.md)
 - [Pool Operations Tutorial](docs/tutorial/intro.md)
 - [High level overview](#overview)
-- [CStor Resource Organisation](docs/developer-guide/cstor-pool-docs.md)
+- [CStor Resource Organisation](docs/developer-guide/cstor-pool.md)
```

Comment:
we can add the direct links here for better visibilty  
- [CStor Pool Resource Organisation](docs/developer-guide/cstor-pool.md)
- [CStor Volume Resource Organisation](docs/developer-guide/cstor-volume.md)

Reply:
One has to go through first the cStor pool always and then the cStor volume. 
Also -- putting more links in the main page can make it look crowded.